.github/workflows/main.yml

```
@@ -0,0 +1,80 @@
+        description: The version name of the release (e.g. 1.1.5)
+        required: true
+        default: ''
+jobs:
+  tag:
+    name: Generate new release package
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ github.event.inputs.version }}
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)